code/modules/mob/living/carbon/alien/alien.dm

```diff
@@ -114,7 +114,9 @@ Des: Removes all infected images from the alien.
 	return
 
 /mob/living/carbon/alien/canBeHandcuffed()
-	return 1
+	if(num_hands < 2)
+		return FALSE
+	return TRUE
 
```

Comment:
Quality change

Reply:
Xenos have limbs, it's natural to expect someone may make them dismemberable down the line.